src/scss/tools/_typography.scss

```diff
@@ -53,6 +53,13 @@
   color: $level-3-body-and-headings;
 }
 
+@mixin body-strong {
```

Comment:
Can't we use just font-weight: 700? Instead of creating a mixin?

Reply:
In Frontify we have this typography, I add because was missing, but I can remove and add the font-weight if you want, but I think we will use a lot of this mixin.

Here is the documentation of that: 
[link](https://company-160717.frontify.com/document/271391#/foundations/typography)

Reply:
We should not have random `font-*` anywhere in the code. All possible combinations should be defined by typography section for consistency.